tests/test_errors.py

```diff
@@ -9,11 +9,32 @@
 
 import pytest
 from flask import Flask
+from werkzeug.exceptions import UnsupportedMediaType
 
+from flask_resources.errors import HTTPJSONException, \
+    UnsupportedMediaTypeRESTError, create_error_handler
 from flask_resources.resources import CollectionResource, Resource, ResourceConfig
 
 
 # These classes are in the file because they are under test too
+class CustomUnsupportedMediaType(HTTPJSONException):
+
+    code = 415
+    description = "My custom exception."
+
+
+class CustomErrorMapConfig(ResourceConfig):
+    """Custom resource configuration."""
+
+    item_route = "/custom/<id>"
+    list_route = "/custom/"
+    error_map = {
+        UnsupportedMediaTypeRESTError: create_error_handler(
+            lambda e: CustomUnsupportedMediaType()
```

Comment:
When there's no transformation logic, shouldn't it just be:

```python
{
     # no mapper function
     UnsupportedMediaTypeRESTError: create_error_handler(CustomUnsupportedMediaType),
     # mapper function
     UnsupportedMediaTypeRESTError: create_error_handler(lambda e: CustomUnsupportedMediaType(e.description) ),
}
```

I.e. no lambda expression.

Reply:
I can implement that yes! I started simply by expecting always one map function.